code/style.css

```diff
@@ -1,7 +1,309 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  min-height: 100vh;
+  background:linear-gradient(#F3C5C5, #C1A3A3, #886F6F, #694E4E);
+  font-family:Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+/* --- HEADER STYLING  --- */
+.image-header {
+  width: 100vw;
+  height: 50vh;
+  background-image: url('/images/hero.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  font-family: 'Sedgwick Ave', cursive;
+}
+
+.image-header .fab {
+  color:rosybrown;
+  padding: 2px;
+  cursor:pointer;
+} 
+
+.header-title {
+  font-size: 36px;
+  display: flex;
+  justify-content: center;
+}
+
+.header-title a {
+  color:mediumpurple;
+}
+
+.inspo {
+  display:flex;
+  justify-content: center;
+  text-align: center;
+  font-size: 16px;
+  color: black;
+}
+
+.inspo:hover {
+  transform: scale(1.4);
+  cursor: pointer;
+}
+
+.header-text {
+  font-size: 14px;
+  color:mediumvioletred;
```

Comment:
I would like a little baby space here between color: and mediumvioletred :)

Reply:
I really love that you're a bit of a details nerd because I'm too! But for this little detail I forgot the space... haha. Thanks for letting me know! :)